app/assets/stylesheets/app/dashboard.scss

```diff
@@ -47,17 +43,52 @@
 
   &--head {
     @extend .matrix__user;
+    height: 55px;
     flex: 0 0 65px;
     padding: 0;
-    padding-top: 12px;
   }
 }
 
 .matrix__user-img {
+  display: block;
```

Comment:
Properties should be ordered border, border-radius, display, max-height, max-width